.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "semi": false,
+  "singleQuote": true
```

Comment:
Can we add "tabWidth": 4? (or something else)
Also do we want to add "trailingComma" property?

Reply:
I like `trailingComma` a lot, and my personal preference on `tabWidth` is `2`.